.github/workflows/build_test.yml

```diff
@@ -250,10 +250,51 @@ jobs:
             .coverage
             coverage.xml
 
+  test-lambda:
+    name: Test Lambda linux-${{ matrix.python-version }}-${{ matrix.cloud-provider }}
+    needs: build
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ["3.8", "3.9", "3.10"]
```

Comment:
Do we need 3.11? Similar question on docker files.

Reply:
aws runtime interface is not yet available for 3.11

Reply:
looks like it was added in the last two months. I'm adding 3.11 support as well